src/peft/tuners/lora/config.py

```diff
@@ -199,6 +293,9 @@ class LoraConfig(PeftConfig):
         eva_config (`Optional[EvaConfig]`):
             The configuration of EVA. At a minimum the dataset argument needs to be set (use the same dataset as for
             finetuning).
+        corda_config (`Optional[CordaConfig]`):
```

Comment:
Could you please also update the docstring and the help of `init_lora_weights` to include `"corda"`?